scripts/tidlers.lua

```diff
@@ -0,0 +1,17 @@
+--Toggle idlers count
+--[[=begin
+
+tidlers
+=======
+Toggle between all possible positions where the idlers count can be placed.
+
+=end]]
+-- see also the enum: df.d_init_idlers
```

Comment:
It would be nice to use enum names here (and even better to use/implement something like ENUM_NEXT_ITEM in lua).